argo-event/templates/eventsource.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: argoproj.io/v1alpha1
+kind: EventSource
+metadata:
+  name: {{ .Values.name }}
+  {{- with .Values.labels }}
+  labels:
+    {{ toYaml . }}
+  {{- end }}
+spec:
+  eventBusName: {{ .Values.name }}
+  {{- with .Values.eventsensor.config }}
+  replicas: {{ .replicas | default 1 }}
+  {{- end }}
+  service:
+    ports:
+      - port: {{ .Values.eventsource.config.servicePort | default 12051 }}
+        targetPort: 12051
```

Comment:
configurable instead of hardcode? (with default)

Reply:
This is used for the eventsource-svc to connect with eventsource-pod, I think hardcode ok